servicemanager/serviceresolver.py

```diff
@@ -28,9 +29,19 @@ def resolve_services_from_array(self, services):
                 services_to_start.remove(not_start)
         return services_to_start
 
+    def _find_assets_service_name(self, service_name):
+        assets_service_name = ''
+        for service_name in self.application.services:
+            if self.application.services[service_name]['type'] == 'assets':
```

Comment:
can we make these strings constants? and perhaps make generic, for example the function could be "find_assets_of_type(self, service_name, type):" this can then be re-used...